### PR TITLE
Modular Computer Program Availability Buffs

### PR DESCRIPTION
**Pull request by ParadoxSpace**
- rscadd: "Some Engineering programs have been re-added to function on wristbounds."
  - rscadd: "All computers can now run Cargo Order."
  - rscadd: "Everyone now gets a Crew Manifest and News Browser on their computers."

### CODE REVIEW

**Comment by ParadoxSpace** [comment]
!review

File: code/modules/modular_computers/file_system/programs/generic/ntsl2_interpreter.dm [hunk 1]
```diff
@@ -3,6 +3,7 @@
 	filedesc = "NTSL2+ Interpreter"
 	extended_desc = "This program is used to run NTSL2+ programs."
 	program_icon_state = "generic"
+	usage_flags = PROGRAM_CONSOLE | PROGRAM_LAPTOP | PROGRAM_TABLET | PROGRAM_STATIONBOUND | PROGRAM_WRISTBOUND
```
Comment (Arrow768) [inline code/modules/modular_computers/file_system/programs/generic/ntsl2_interpreter.dm:6]: Undocumented change.
```suggestion
	usage_flags = PROGRAM_CONSOLE | PROGRAM_LAPTOP | PROGRAM_TABLET
```
If you want to use NTSL Programs, you should imho get a more capable device than a wristbound.

File: code/modules/modular_computers/file_system/programs/generic/news_browser.dm [hunk 2]
```diff
@@ -3,6 +3,7 @@
 	filedesc = "NTNet/ExoNet News Browser"
 	extended_desc = "This program may be used to view and download news articles from the network."
 	program_icon_state = "generic"
+	usage_flags = PROGRAM_CONSOLE | PROGRAM_LAPTOP | PROGRAM_TABLET | PROGRAM_STATIONBOUND | PROGRAM_WRISTBOUND
```
Comment (Arrow768) [inline code/modules/modular_computers/file_system/programs/generic/news_browser.dm:6]:
```suggestion
	usage_flags = PROGRAM_ALL
```

File: code/modules/modular_computers/file_system/programs/engineering/_engineering.dm [hunk 3]
```diff
@@ -11,6 +11,7 @@
 	required_access_download = access_ce
 	requires_ntnet = TRUE
 	network_destination = "power monitoring system"
+	usage_flags = PROGRAM_CONSOLE | PROGRAM_LAPTOP | PROGRAM_TABLET | PROGRAM_STATIONBOUND | PROGRAM_WRISTBOUND
```
Comment (Arrow768) [inline code/modules/modular_computers/file_system/programs/engineering/_engineering.dm:14]:
```suggestion
	usage_flags = PROGRAM_ALL
```

File: code/modules/modular_computers/file_system/programs/engineering/_engineering.dm [hunk 4]
```diff
@@ -40,6 +41,7 @@
 	extended_desc = "This program provides visual interface for station's alarm system."
 	requires_ntnet = TRUE
 	network_destination = "alarm monitoring network"
+	usage_flags = PROGRAM_CONSOLE | PROGRAM_LAPTOP | PROGRAM_TABLET | PROGRAM_STATIONBOUND | PROGRAM_WRISTBOUND
```
Comment (Arrow768) [inline code/modules/modular_computers/file_system/programs/engineering/_engineering.dm:44]:
```suggestion
	usage_flags = PROGRAM_ALL
```

File: code/modules/modular_computers/file_system/programs/civilian/cargo_order.dm [hunk 5]
```diff
@@ -5,7 +5,7 @@
 	size = 10
 	requires_ntnet = TRUE
 	available_on_ntnet = TRUE
-	usage_flags = PROGRAM_LAPTOP | PROGRAM_TELESCREEN | PROGRAM_CONSOLE
+	usage_flags = PROGRAM_CONSOLE | PROGRAM_LAPTOP | PROGRAM_TABLET | PROGRAM_STATIONBOUND | PROGRAM_TELESCREEN | PROGRAM_WRISTBOUND
```
Comment (Arrow768) [inline code/modules/modular_computers/file_system/programs/civilian/cargo_order.dm:8]:
```suggestion
	usage_flags = PROGRAM_ALL_REGULAR
```

File: code/modules/modular_computers/file_system/programs/generic/ntsl2_interpreter.dm (hunk 1 above)
Comment (Karolis2011) [inline code/modules/modular_computers/file_system/programs/generic/ntsl2_interpreter.dm:6]: Why are you adding restrictions? Also this isn't mentioned in changelog. And why pAIs are excluded?

File: code/modules/modular_computers/file_system/programs/generic/manifest.dm [hunk 6]
```diff
@@ -3,6 +3,7 @@
 	filedesc = "Crew Manifest"
 	program_icon_state = "generic"
 	extended_desc = "This program is used for viewing the crew manifest."
+	usage_flags = PROGRAM_ALL
```
Comment (Karolis2011) [inline code/modules/modular_computers/file_system/programs/generic/manifest.dm:6]: This is unecessery

File: code/modules/modular_computers/file_system/programs/generic/news_browser.dm (hunk 2 above)
Comment (Karolis2011) [inline code/modules/modular_computers/file_system/programs/generic/news_browser.dm:6]: pAI can't access news? Just do not add restrictions.

File: code/modules/modular_computers/file_system/programs/engineering/_engineering.dm (hunk 4 above)
Comment (Karolis2011) [inline code/modules/modular_computers/file_system/programs/engineering/_engineering.dm:44]: Why add restrictions? Also please do not list every type and use agregte type `PROGRAM_ALL_REGULAR` with agregite type `PROGRAM_STATIONBOUND`

File: code/modules/modular_computers/file_system/programs/civilian/cargo_order.dm (hunk 5 above)
Comment (Karolis2011) [inline code/modules/modular_computers/file_system/programs/civilian/cargo_order.dm:8]: Use agregite types.

File: code/modules/modular_computers/file_system/programs/generic/manifest.dm (hunk 6 above)
Comment (Arrow768) [reply code/modules/modular_computers/file_system/programs/generic/manifest.dm:6]: Imho a explicit definition of the usage_flags for the modular computer programs is useful (even in the default case) so its easy to see what flags a application has.

File: code/modules/modular_computers/file_system/programs/generic/news_browser.dm (hunk 2 above)
Comment (Arrow768) [reply code/modules/modular_computers/file_system/programs/generic/news_browser.dm:6]: Already addressed in my review.

File: code/modules/modular_computers/file_system/programs/engineering/_engineering.dm (hunk 4 above)
Comment (Arrow768) [reply code/modules/modular_computers/file_system/programs/engineering/_engineering.dm:44]: Already mentioned in the above review.

File: code/modules/modular_computers/file_system/programs/civilian/cargo_order.dm (hunk 5 above)
Comment (Arrow768) [reply code/modules/modular_computers/file_system/programs/civilian/cargo_order.dm:8]: Already mentioned in the above review.

File: code/modules/modular_computers/file_system/programs/generic/ntsl2_interpreter.dm (hunk 1 above)
Comment (ParadoxSpace) [reply code/modules/modular_computers/file_system/programs/generic/ntsl2_interpreter.dm:6]: This was introduced in a previous PR.

**Pull request merged**